student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(age, name, parent_permission: true)
+    @classroom = classroom
+  end
```

Comment:
- [ ] in the initialize method apart from `classroom` you also need to pass `age`, `name` and `parent_permission` as arguments to make it work.